var/spack/repos/builtin/packages/votca/package.py

```diff
@@ -39,7 +39,8 @@ class Votca(CMakePackage):
     depends_on("expat")
     depends_on("fftw-api@3")
     depends_on("eigen@3.3:")
-    depends_on("boost")
+    depends_on("boost@1.71:1.84+filesystem+system+regex+timer")
```

Comment:
Why is there an upper bound?

Reply:
I thought we fixed that in https://github.com/votca/votca/pull/1123

Reply:
boost/filesystem/convenience.hpp was removed in 1.85

Reply:
I don't believe that's made it into a release yet. I suppose I should bound it on @2023:2024